app/test_main.py

```diff
@@ -1,3 +1,49 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_age",
+        [
+            pytest.param(14, 14, [0, 0],
+                         id="Up to 15 cat/dog years give 0 human year"),
+            pytest.param(15, 15, [1, 1],
+
+                         id="First 15 cat/dog years give 1 human year"),
+            pytest.param(23, 23, [1, 1],
+                         id="24 cat/dog years should be equal 1 human years"),
```

Comment:
You check 23  cat/dog years but id secription about 24?

Reply:
replase this one with test that you had but delete - pytest.param(27, 28, [2, 2]